graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -532,6 +532,28 @@ def test_graph_relationshipProperties_stream_without_arrow_separate_property_col
     assert {e for e in result["relY"]} == {5, 6, 7}
 
 
+@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 2, 0))
+def test_graph_relationships_stream_without_arrow(gds_without_arrow: GraphDataScience) -> None:
+    G, _ = gds_without_arrow.graph.project(GRAPH_NAME, "*", "REL")
+
+    result = gds_without_arrow.beta.graph.relationships.stream(G, ["REL"])
+
+    assert list(result.keys()) == ["sourceNodeId", "targetNodeId", "relationshipType"]
+    assert {e for e in result["sourceNodeId"]} == {0, 1}
+    assert {e for e in result["targetNodeId"]} == {1, 2}
+
```

Comment:
we did not depend on the neo4j ids before. As we are testing multiple version combinations on the same neo4j db, these fail here. 

@soerenreichardt Instead we should get the neo-ids via a Cypher query and uses these for the expected ids

Reply:
fixing a bug https://github.com/neo-technology/graph-analytics/pull/5952

Reply:
waiting for fix on server-side to be merged

Reply:
Yes that should be the reason for the flaky tests here